app/services/reporting/total_user_count_report.rb

```diff
@@ -60,13 +60,13 @@ def total_user_count
 
     def verified_user_count
       Reports::BaseReport.transaction_with_timeout do
-        Profile.where(active: true).where('activated_at <= ?', end_date).count
+        Profile.where(active: true).where('verified_at <= ?', end_date).count
       end
     end
 
     def new_verified_user_count
       Reports::BaseReport.transaction_with_timeout do
-        Profile.where(active: true).where(activated_at: current_month).count
+        Profile.where(active: true).where(verified_at: current_month).count
```

Comment:
do we need any corresponding spec updates for this?

Reply:
The current spec is covering most of the cases which works for verified_at column.  if i can add a case where user is deactivated/reproofing will the values be different for activated_at and verified_at?

Reply:
yeah I think that would be good! it would help us catch regressions

Reply:
yes in particular we need to add the following edge cases

- profile has verified_at before the current month range but updated_at is in the current month range (ex. reactivated accounts); profiles in this case should not be counted towards the current month
- profile has both verified_at & updated_at in the current month (ex. newly activated/verified accounts), there should be cases like this in the spec files already